packages/launchpad/src/encoding.ts

```diff
@@ -5,24 +5,30 @@ import { Uint53 } from "@cosmjs/math";
 import { Msg } from "./msgs";
 import { StdFee } from "./types";
 
-function sortJson(json: any): any {
-  if (typeof json !== "object" || json === null) {
-    return json;
+function sortedObject(obj: any): any {
```

Comment:
Suggest naming this `sortObject`.

Reply:
This follows the idea that "sort" is a mutating operation while "sorted" returns a sorted copy of the object. This way it is done [in Python](https://discuss.codecademy.com/t/what-is-the-difference-between-sort-and-sorted/349679) and [in Swift](https://stackoverflow.com/questions/40621076/whats-the-difference-between-sorted-and-sort-in-array) and I liked the concept. In JavaScript "sort" is a mutating operation as well, which also returns the sorted object for no clear reason.

Reply:
Fine with me. For the record I quite like the approach of using the same name with (non-mutating) or without (mutating) a trailing underscore which kind of hints that mutation is weird and bad.
Eg https://pytorch.org/docs/stable/tensors.html#torch.Tensor.abs

Reply:
:) better than the "i" prefix of bn.js for sure